### PR TITLE
neogit plugin for nvcommunity

### PR DESCRIPTION
**Pull request by ambaradan**
The configuration has been tested several times on new installations and has given no problems.

### CODE REVIEW

File: lua/nvcommunity/git/neogit/init.lua
Comment (lucario387) [inline lua/nvcommunity/git/neogit/init.lua:1]: stylua this file please

Comment (ambaradan) [reply lua/nvcommunity/git/neogit/init.lua:1]: I apologize, but I really don't know where to start to do this. How should I do it? Any pointers?

Comment (lucario387) [reply lua/nvcommunity/git/neogit/init.lua:1]: from the cli, do `stylua lua/nvcommunity/git/neogit`

Comment (lucario387) [reply lua/nvcommunity/git/neogit/init.lua:1]: If not I'll just do it myself and merge the PR

Comment (ambaradan) [reply lua/nvcommunity/git/neogit/init.lua:1]: Okay, if you can do it for this file in the meantime, I'll see how to do it for the next ones.
Thank you

**Pull request merged**